test/framework/utils/utils.go

```diff
@@ -14,14 +14,30 @@
 package utils
 
 import (
+	appsV1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
```

Comment:
These are already defined in framework/utils/constants

Reply:
Oh okay, will make this minor change then